### DISCUSSION TITLE
请问，在2.x版本怎么绑定view

### DISCUSSION THREAD

**Discussion by news-ttt** [antvis/X6#3415] [Q&A]
绑定view是不是重新渲染页面时，页面不会闪烁一下，有没有2.x绑定view的示例啊
![image](https://user-images.githubusercontent.com/101792817/226862640-9815d21b-4793-48d9-b90c-2423938737b4.png)

**Comment by NewByVector** [comment]
具体什么使用场景呢？

**Reply by news-ttt** [reply]
我用的是react, 1.x的时候呢，是引入了usePortal，然后给他了uniqViewId，node绑定时直接绑定了那个id，然后页面上引用了uniqViewI返回的组件，2.x没有usePortal, 有没有方法替代他

我页面的数据都是通过后端计算的，样式也是自定义的，然后没新增一个或者更改一个，页面都会闪烁，主要需求就是解决这个闪烁的问题，1.x时用usePortal时没有闪烁

**Reply by NewByVector** [reply]
可以自己计算数据 diff，然后按需渲染。
